egs/wsj/s5/utils/queue.pl

```diff
@@ -82,6 +87,7 @@ ()
    "and change its behavior.  Otherwise it uses qstat to work out when the job finished\n" .
    "Options:\n" .
    "  --config <config-file> (default: $config)\n" .
+   "  --early-exit <true | false> (default: false). If true, if one of the jobs in the array fails, all the jobs submitted will be deleted and the script will exit\n" .
```

Comment:
please break the line here to avoid very long line [see how it's done for --mem].
And let's have this default to true; it's normally what we want.